dragonchain/lib/database/redis.py

```diff
@@ -161,6 +161,16 @@ def _initialize_redis(host: str, port: int, wait_time: int = 30) -> redis.Redis:
     )
 
 
+async def lrange_async(key: str, start: int, end: int) -> List:
```

Comment:
Probably need a decode flag here

Reply:
imma kill this and replace with set stuff